deploy/catalog/Chart.yaml

```diff
@@ -1,3 +1,3 @@
-name: catalog
-description: Kubernetes incubator Service Catalog deployment Helm chart.
+name: kubernetes-service-catalog
```

Comment:
In some unachievable ideal, maybe it's globally unique, but that's in no way an actual technical constraint. As we hack along, we tend to `helm install deploy/catalog` where `deploy/catalog` is a local directory. In "real life," charts are archives fetched over HTTPS from a repository. It's virtually the same as what any other package manager does. And as with any other package manager, multiple repositories _could_ provide different packages having the same name...

All this said, the "disambiguation" (by specifying "kubernetes") here is probably not required. After all-- Helm doesn't install packages to anywhere _but_ kubernetes, so I agree with @vaikas-google here.